app/service-cli.cpp

```diff
@@ -21,8 +21,14 @@ int main(int argc, char *argv[]) {
   std::string input = std_input.str();
   using marian::bergamot::Response;
 
+  marian::Ptr<marian::Options> responseOptions = marian::New<marian::Options>();
```

Comment:
@jerinphilip Should the app `service-cli-bytearray` also be updated?

Reply:
When I synchronize with main, yes. Expecting CI to complain when https://github.com/browsermt/bergamot-translator/pull/93 is merged.

Reply:
So the printing code is pulled into the library to avoid duplication. This can be eventually extended maybe to output in a more readable format and to programmatically check cases (no alignments, yes alignments, no quality, quality). Both service-cli and service-cli-bytearray reuses this bit.

Stopping at a minimum invasive edit for now which eliminates duplication.